benches/linalg/schur.rs

```diff
@@ -1,58 +1,82 @@
-use na::{Matrix4, Schur};
+use na::Schur;
 
 fn schur_decompose_4x4(bh: &mut criterion::Criterion) {
-    let m = Matrix4::<f64>::new_random();
-    bh.bench_function("schur_decompose_4x4", move |bh| {
-        bh.iter(|| std::hint::black_box(Schur::new(m.clone())))
+    bh.bench_function("schur_decompose_4x4", |bh| {
+        bh.iter_batched(
+            || crate::reproducible_smatrix::<f64, 4, 4>(),
```

Comment:
why is the reproducible matrix called here? I'm not so familiar with the Schur decomposition, but from a cursory glance at wikipedia, any square real matrix should have one. Same question for the other instances of the test below.